deploy/_filter_files.py

```diff
@@ -11,7 +11,7 @@ def main(argv: List[str]) -> None:
     if file_extension.startswith("."):
```

Comment:
This file is used by other steps as well as typechecking, so will affect those too - is that deliberate?

Reply:
Yeah this is meant to stop it forcing people to rewrite a whole file as per ruff/black guidelines for code in the older modules, as our discussion on thursday/friday.